src/Client/AddClient.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace RialtoWebService\Client;
+
+use RialtoWebService\ClassMap;
+use RialtoWebService\Result\AddOrdersResult;
+use RialtoWebService\ServiceType\Add;
+use RialtoWebService\StructType\AddRialtoOrders;
+use RialtoWebService\StructType\RialtoWebOrderList;
+use WsdlToPhp\PackageBase\AbstractSoapClientBase;
+
+class AddClient
+{
+    const LIVE_WSDL_URL = 'https://rws.redarrow.co.uk/service.asmx?wsdl';
+    const TEST_WSDL_URL = 'http://test.rws.redarrow.co.uk/service.asmx?wsdl';
+
+    /** @var string */
+    private $username;
+
+    /** @var string */
+    private $password;
+
+    /** @var string */
+    private $wsdlUrl;
+
+    public function __construct(string $username, string $password, bool $testMode = false)
```

Comment:
Optional params are smelly. Better to inject the URL instead of an optional flag.

Reply:
ok!